### ISSUE TITLE
new grpc version with ChannelCredentials not updated in homebrew

### ISSUE THREAD

**Issue by Rastusik** [grpc/homebrew-grpc#72]
On 16.12.2015, the grpc lib was updated with mandatory credentials usage. When I try to compile the php grpc extension from source, I'm getting these errors:

```
./channel_credentials.h:57:3: error: unknown type name 'grpc_channel_credentials'; did you mean 'grpc_server_credentials'?
  grpc_channel_credentials *wrapped;
  ^~~~~~~~~~~~~~~~~~~~~~~~
  grpc_server_credentials
```

I'm trying to compile the php extension from grpc source, using the libs from homebrew grpc, which aren't up to date

**Comment by murgatroid99** [comment]
The latest published version of gRPC (anywhere) is 0.11. If you want to build the PHP extension from source against a published version of gRPC, you will have to use the release-0_11 branch of the grpc repository.

Version 0.12, which includes the changes you mention, will be published very soon.

**Comment by stanley-cheung** [comment]
Please use PECL to install the gRPC PHP extension.
